syntax = "proto3";

package notifier_um;

service NotifierUm {
  rpc RequestEmail (Request) returns (Reply) {}
}

message Request {
  int64 user_id = 1;
}

message Reply {
  string email = 1;
}